src/submission/parseErrors/index.jsx

```diff
@@ -126,6 +127,30 @@ class ParseErrors extends Component {
   }
 }
 
+function errorResponseParser(errorResponse, uliNeeded) {
+  let apiErrorMessages =[]
+
+  errorResponse.forEach(function(errorsFound) {
+    if(errorsFound && errorsFound.errorMessages)
+    for (const [index, errorMessage] of errorsFound.errorMessages.entries()) {
+      let inputContent = errorMessage.inputValue
+
+      if (!errorMessage.inputValue) inputContent = <em>(blank)</em>
+ 
+      apiErrorMessages.push(
+        <tr key={index}>
```

Comment:
This key should be a concatenation of the indices of the outer loop and the inner loop to be unique.
So add a second argument `i` to the function provided to `forEach` on line 133 and then change this to:
```suggestion
        <tr key={`${i}${index}`}>
```